sycl/include/CL/sycl/detail/scheduler/commands.hpp

```diff
@@ -176,11 +184,12 @@ class ReleaseCommand : public Command {
       : Command(CommandType::RELEASE, std::move(Queue)), MAllocaCmd(AllocaCmd) {
   }
 
-  void printDot(std::ostream &Stream) const override;
+  void printDot(std::ostream &Stream) const override final;
```

Comment:
```suggestion
  void printDot(std::ostream &Stream) const final;
```
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c128-virtual-functions-should-specify-exactly-one-of-virtual-override-or-final

Please, apply to other instances of "override final".

Reply:
Done.